app/utils/duration.rb

```diff
@@ -76,7 +80,7 @@ def format_date(value)
   end
 
   def date_only?(value)
```

Comment:
Moved this method to make it available outsite of duration, but kept original method in case any of the wagons somehow depend on this (e.g. with ```send```)

Reply:
Good thinking. I would defend the removal against anyone, since "thou shalt not `send` for private methods". Then again, someone might have solved a problem this way and this gives an nice upgrade-path. One could add a deprecation warning (https://devdocs.io/rails~4.2/module#method-i-deprecate) to instruct the caller about your new and better way.

Reply:
👍